.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: go
+
+go:
+  - 1.7
+  - 1.8
```

Comment:
You could target the latest patch versions of each minor version with:

```YML
go:
  - 1.7.x
  - 1.8.x
```

Otherwise you're going to get 1.7(.0), 1.8(.0) instead of 1.7.5, 1.8.1.

Reference: https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use.